202311 lms-refactor (#844)
utelecon/utelecon.github.io: src/pages/en/itc_lms/lecturers/index.md

---
title: How to use cources' functions (for instructors)
breadcrumb:
  title: Cources (for instructors)
redirect_from:

>   - /en/lms_lecturers

Reviewer: ```suggestion
  - /en/lms_lecturers/
```

  - /en/lms_lecturers/timetable
---

In ITC-LMS, each subject carried out in a term or a semester is called a "course."

> You would be using course functions in most cases of using ITC-LMS.

Reviewer 2: ```suggestion
In most cases, you are likely to use this course feature in ITC-LMS.
```
would be using に違和感があったのでこうしてみました（改善の余地はまだあるかと思います）
